agent/proxycfg/state.go

```diff
@@ -16,12 +16,13 @@ import (
 )
 
 const (
-	coalesceTimeout       = 200 * time.Millisecond
-	rootsWatchID          = "roots"
-	leafWatchID           = "leaf"
-	intentionsWatchID     = "intentions"
-	serviceIDPrefix       = string(structs.UpstreamDestTypeService) + ":"
-	preparedQueryIDPrefix = string(structs.UpstreamDestTypePreparedQuery) + ":"
+	coalesceTimeout                  = 200 * time.Millisecond
+	rootsWatchID                     = "roots"
+	leafWatchID                      = "leaf"
+	intentionsWatchID                = "intentions"
+	serviceIDPrefix                  = string(structs.UpstreamDestTypeService) + ":"
+	preparedQueryIDPrefix            = string(structs.UpstreamDestTypePreparedQuery) + ":"
+	DefaultPreparedQueryPollInterval = 30 * time.Second
```

Comment:
Is this intentionally Public? If so a comment about who else cares would be good!

Reply:
At the time I was thinking other things would care. Turns out nothing needs to.